css/style.css

```diff
@@ -724,6 +770,12 @@ ul.chars{
     margin: 0px 4px;
 }
 
+.subnav:hover {
+    border-bottom: 0.1vh solid white;
```

Comment:
0.1vh is going to be 1px or for a massive number of screens, you should just use 1px to draw a cleaner line and make sure the browser doesn’t try to draw a fractional one